examples/tests/trinity.py

```diff
@@ -33,7 +33,7 @@ def setup(self):
         """
         tarball_path = self.get_data_path(self.params.tarball)
         archive.extract(tarball_path, self.srcdir)
-        self.srcdir = os.path.join(self.srcdir, 'trinity-1.4')
+        self.srcdir = os.path.join(self.srcdir, 'trinity-2015-01-28')
```

Comment:
I see a problem here, if you take the snapshot from another day, with avocvado, the date will be different.